src/components/Card.tsx

```diff
@@ -83,6 +93,11 @@ export const Card = (props: CardProps) => {
     setTextOutputHeight(target.scrollHeight);
   };
 
+  const dynamicClasses = [
+    styles.card,
+    playAnimation ? styles.flipVerticalRight : "",
+  ].join(" ")
```

Comment:
I couldn't find it in the design, but initially I was thinking of making two animations - right flip and left flip - which would play depending on which side was displayed initially. If you think I should add it, I will.